lib/crawler_worker/keyword/crawler_worker.ex

```diff
@@ -15,7 +15,8 @@ defmodule CrawlerWorker.Keyword.CrawlerWorker do
 
     case GoogleClient.crawl(keyword.name) do
       {:ok, html} ->
-        Keywords.mark_as_completed(keyword, %{html: html})
+        attrs = KeywordExtractor.parse(html)
+        Keywords.mark_as_completed(keyword, Map.merge(attrs, %{html: html}))
```

Comment:
should we have some tests to make sure the 
- HTML is parseable (this is done in the test/crawler/keyword/keyword_extractor_test.exs test file)
- The result is stored in the database

The current test doesn't mention whether the extracted information could be stored in the database yet or not.

Reply:
@andyduong1920 Good point, I will add those unit tests. 

For the task's ACs, I need to adjust a little bit to make it more "backend" concern 💪 smt like:
- Backend -> updating schema, extracting and storing the result.
- UI -> controller, route, view, and interaction.
What do you think?

Reply:
Ah, FYI, in the web project, the Backend task mean

- All database stuff, migration, field, store...
- All controller setup
- Render the simple view

And the UI task means
- Polish the UI which contains only HTML/CSS or Elixir helper code to make the UI look nicer, the controller, route or plug won't be in the UI task


So the suggestion for you now is:

- Add the `Part 1` suffix into this PR, so this PR will be the part 1 for the #20 which will parse and store the data into the database
- We will open a second PR `with Part 2` for the #20 task, which will set up the controller, route, and render the simple view, of course, it will contain the controller test and simple system test.

Is that clear @vnntsu ?

Reply:
Oh, that's clear indeed. Thanks! I'll create part 2 🙏 
But I have this concern, I don't really focus on the frontend, bad and nice-looking UI are both tough for me. What do you think about we leave UI task for now? since based on the suggestion, we're quite done with the screen and the feature works well btw.

Reply:
> What do you think about we leave UI task for now? since based on the suggestion, we're quite done with the screen and the feature works well btw.

As a reviewer, we can't make this decision, this is the agreement between you and your manager @vnntsu 😄 like you would like to reduce the work on the UI part, please communicate to your manager, and inform us the result @vnntsu cc @Nihisil